crates/oxc_linter/src/snapshots/escape_case.snap

```diff
@@ -0,0 +1,53 @@
+---
+source: crates/oxc_linter/src/tester.rs
+expression: escape_case
+---
+  ⚠ eslint-plugin-unicorn(escape-case): Use uppercase characters for the value of the escape sequence.
+   ╭─[escape_case.tsx:1:1]
+ 1 │ const foo = "\xAab\xaab\xAAb\uAaAab\uaaaab\uAAAAb\u{AaAa}b\u{aaaa}b\u{AAAA}";
```

Comment:
is it worth reporting the error on the exact two chars that there is the issue?

Reply:
Eslint does the same thing. I don't think there's much difference.